### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by YegorKochetkov**
[DEMO LINK](https://YegorKochetkov.github.io/react_static-list-of-todos/)

### CODE REVIEW

**Review by ivan-kadykalo — APPROVED** [review]
Looks like everithing is fine

File: src/components/TodoInfo/TodoInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+export const TodoInfo: React.FC<Props> = ({
+  title,
+  status,
+  user,
+}) => (
+  <>
+    <p className="todo-info">
+      <div className="todo-info__item">
+        <span className="todo-info__title">Task:</span>
+        {` ${title}`}
```
Comment (MaksimusCRS) [inline src/components/TodoInfo/TodoInfo.tsx:21]:
```suggestion
        {title}
```

File: src/components/TodoInfo/TodoInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      </div>
+      <div className="todo-info__item">
+        <span className="todo-info__title">Status:</span>
+        {` ${status ? 'Done' : 'Not ready'}`}
```
Comment (MaksimusCRS) [inline src/components/TodoInfo/TodoInfo.tsx:25]:
```suggestion
        {status ? 'Done' : 'Not ready'}
```

File: src/components/UserInfo/UserInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+type Props = {
+  user: User | null,
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <p className="user-info">
+    <div className="user-info__item">
+      <span className="user-info__title">Assigned:</span>
+      {` ${user?.name}`}
```
Comment (MaksimusCRS) [inline src/components/UserInfo/UserInfo.tsx:13]:
```suggestion
      {user?.name}
```

**Review by MaksimusCRS — APPROVED** [review]

**Review by MaksimusCRS — APPROVED** [review]
test

**Pull request closed without merging**